packages/block-library/src/block/edit.js

```diff
@@ -38,6 +45,15 @@ import { unlock } from '../lock-unlock';
 const { useLayoutClasses } = unlock( blockEditorPrivateApis );
 const fullAlignments = [ 'full', 'wide', 'left', 'right' ];
 
```

Comment:
The following method should also check for block support of `__experimentalConnections` but for ease of testing without having to merge with connections PRs just checking for attributes for now.